--- origsrc/ibus-input-pad-1.4.99.20140916/setup/Makefile.am	2014-09-15 09:00:16.000000000 -0500
+++ src/ibus-input-pad-1.4.99.20140916/setup/Makefile.am	2015-07-23 18:06:35.863456100 -0500
@@ -77,7 +77,7 @@ ibus_setup_input_pad_LDADD = \
 	$(NULL)
 
 resources.c: ibus-setup-input-pad.gresource.xml
-	$(GLIB_COMPILE_RESOURCES) $< --target=$@ --generate-source
+	$(GLIB_COMPILE_RESOURCES) --sourcedir=$(srcdir) --target=$@ --generate-source $<
 
 desktop_in_in_files = ibus-setup-input-pad.desktop.in.in
 desktop_in_files = $(desktop_in_in_files:.in.in=.in)
